src/vm/memory/relocatable.zig

```diff
@@ -126,7 +126,7 @@ pub const Relocatable = struct {
         other: u64,
     ) !Self {
```

Comment:
Here, as a result of your refactor, can we enforce a better typing of the output? like:

```zig
pub fn subUint(
        self: Self,
        other: u64,
    ) MathError!Self {
```